src/tables/Project.sol

```diff
@@ -30,14 +33,22 @@ contract Project is TablelandController, Ownable {
                 "proposalLink text,"
                 "finalReportIPFS text,"
                 "finalReportLink text,"
-                "contributors text",
+                "contributors text,"
+                "active integer,"
+                "eligible integer",
                 _TABLE_PREFIX
             )
         );
     }
 
+    function setProjectTeam(address projectTeam) external onlyOwner{
+        _projectTeam = ProjectTeam(projectTeam);
+    }
+
     // Let anyone insert into the table
-    function insertIntoTable(string memory title, uint256 quarter, uint256 year, uint256 MDP, string memory proposalIPFS, string memory proposalLink, string memory finalReportIPFS, string memory finalReportLink, string memory contributors) external {
+    function insertIntoTable(uint256 projectTeamId, string memory title, uint256 quarter, uint256 year, uint256 MDP, string memory proposalIPFS, string memory proposalLink, string memory finalReportIPFS, string memory finalReportLink, string memory contributors, uint256 active, uint256 eligible) external {
+        //only let projectTeam.projectTeamCreator insert
+        require(_projectTeam.projectTeamCreator() == msg.sender, "Only ProjectTeamCreator can insert");
```

Comment:
I might be wrong, but this might need to be the project team manager v.s. creator, since the creator will be the moondao treasury?